user-guide/Basic_Functionality/Visio/linking_shapes/Linking_a_shape_to_an_Automation_script.md

```diff
@@ -8,7 +8,7 @@ When a shape is linked to an Automation script, by default this script will be e
 
 > [!NOTE]
 >
-> - For an example, see [Ziine](xref:ZiineDemoSystem) > *Visual Overview Design Examples* view > *[linking > EXE]* page.
+> - For an example, see [Ziine](xref:ZiineDemoSystem) > *Operations* > *Other* > *Visual Overview Design Examples* view > *[linking > EXE]* page.
```

Comment:
I just took a look on ziine, but I can't find any "Other" view under "Operations". Does this still need to be added, or has it been moved elsewhere? 
![image](https://user-images.githubusercontent.com/94605575/211734779-8768350b-2b0f-4607-bef7-344dd8380a7e.png)

Reply:
I'm looking to ziine prod, not ziine dev.

<img width="529" alt="image" src="https://user-images.githubusercontent.com/100204121/211761874-6d053f58-ced0-4717-8e4d-cc136f64e6f8.png">